### DISCUSSION TITLE
Unable to add network monitor to panel

### DISCUSSION THREAD

**Discussion by GenericDataflow** [lxqt/lxqt#2231] [Q&A] [answered]
##### Expected Behavior
Right click panel > Configure panel> add plugin >select network monitor , click add widget >network monitor gets added to my panel

##### Current Behavior
Right click panel > Configure panel> add plugin >select network monitor , click add widget >Nothing happens The Windows stays open and i continue to press add widget to no avail.

##### Possible Solution
See Context

##### Steps to Reproduce (for bugs)

- Steps mentioned in behaviour
- visual representation https://postimg.cc/nX77HW9j

##### Context
After Updating (Pacman -Syu) restarted. Woke up loaded all good 3 hours later no widget no connection.
Started connection manually with if config and for some reason had to unblock wifi via rf-kill. This may not be related but both widget and wifi broke at same time. Creating a new panel yields the same bugs                                       --->

##### System Information
Arch Linux
* Distribution & Version: Arch 5.17.5
* Kernel: 5.17.5-zen1-1-zen
* Qt Version: 5.15.3
* liblxqt Version: 1.1.0-2
* Package version:1.1.0

**Comment by stefonarch** [comment, marked as answer]
`libstatgrab` is installed?

**Reply by GenericDataflow** [reply]
Yes

**Reply by GenericDataflow** [reply]
Oh nice thank you! it must have been corrupt , `pacman -S libstatgrab` it failed integrity check and after reinstall it lets me add the widget again. The best part is now that you have me RTFM i see that its nm-tray thats needed for what I wanted.....And thats broken aswell. Thank you

**Comment by tsujan** [comment]
Not reproducible.
